frontend/src/entities/towers/Tower.js

```diff
@@ -1,9 +1,15 @@
 import Entity from "../Entity";
 
+export const Status = {
+    PENDING: Symbol("pending"),
+    BUILT: Symbol("built"),
+};
```

Comment:
Haven't seen symbols before, are they basically like enums, but useable in js?

Reply:
(At least in this use case)

Reply:
Yea, I did this to basically have it behave like an enum. `Symbol()`, as I've used it, returns globally unique values for each call. It's a bit odd as some things are in js but should get the job done until we transition to Typescript.